mlrun/db/httpdb.py

```diff
@@ -351,6 +351,7 @@ def connect(self, secrets=None):
                 server_cfg.get("force_run_local") or config.force_run_local
             )
             config.function = server_cfg.get("function") or config.function
+            config.ce.mode = server_cfg.get("ce_mode") or config.ce.mode
```

Comment:
If the server here is not ce (`""`), but for some reason the client is (`"lite"` or `"full"`), the config will stay with the ce (`"lite"` or `"full"`). This isn't good. I suggest always overriding with what the server returns.
```suggestion
            config.ce.mode = server_cfg.get("ce_mode", "")
```